server/tests/services/test_hpa_vpa_recommendations.py

```
@@ -0,0 +1,178 @@
+    prior = 0.60
+    assert is_materially_worse(prior * MATERIALLY_WORSE_FACTOR, prior) is True
+    assert is_materially_worse(0.90, prior) is True
+
+
+def test_not_materially_worse_below_the_factor():
+    assert is_materially_worse(0.70, 0.60) is False   # 1.17x
+    assert is_materially_worse(0.60, 0.60) is False   # unchanged
+    assert is_materially_worse(0.30, 0.60) is False   # improved
+
+
+@pytest.mark.parametrize("new,prior", [
```

Ruff (0.16.0)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)